RPLCD/lcd.py

```diff
@@ -450,6 +462,11 @@ def create_char(self, location, bitmap):
         # Restore cursor pos
         self.cursor_pos = pos
 
+    def backlight(self, value):
+        """Turn backlight GPIO on or off."""
+        if self.pins.backlight != None:
+            GPIO.output(self.pins.backlight, bool(value) ^ bool(self.backlight_active_low))
+
```

Comment:
For this to work, we need to initialize the `self._backlight_enabled` variable. I'd do that around line 179, right after `self.numbering_mode = ...`:

```python
self._backlight_enabled = True
```